outputs/otlp_init.go

```diff
@@ -0,0 +1,84 @@
+package outputs
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/falcosecurity/falcosidekick/types"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+	otelresource "go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.20.0"
+)
+
+const (
+	OTLPinstrumentationName    = "falcosidekick.otlp"
```

Comment:
I prefer to set `falco`, as it's the source.

Reply:
+1, done at a7865a5e, s/shot (using the above `docker-compose.yaml`) just for completeness
![image](https://github.com/falcosecurity/falcosidekick/assets/88727/b16cff5d-44bc-4522-9a2b-3bee48559bb5)